### ISSUE TITLE
1.20.1: Bungee discord for the servers that do not have discordSRV installed

### ISSUE THREAD

**Issue by SpectreVoid** [kangarko/ChatControl#2429]
### Are you using MySQL?

Yes

### Are you using BungeeCord/Velocity?

Yes - BungeeCord

### Question

Hey, so I have a waterfall/paper 1.20.1 network. I am having trouble setting up the DiscordSRV using the bungee forward method on the servers that do not have the discordSRV installed. 

My lobby server (without DiscordSRV-related config)

````
standard:
  Format: chat
  Format_Spy: '@noprefix chat'
  Bungee: true
      
Discord: 
  Enabled: false
  Filter_Chat: true
  Remove_Emojis: true
  Webhook: false
  Send_Messages_As_Bot: true
  Forward_From_Bungee: false
````

And the config for my survival server with the discordSRV installed. (It works on this server, but not the other ones that have the same settings as the lobby)

````
standard:
  Format: chat
  Format_Spy: '@noprefix chat'
  Bungee: true
  Discord_Channel_Id: 1140759262906286171
      
Discord:
  Enabled: true
  Filter_Chat: false
  Remove_Emojis: true
  Webhook: true
  Send_Messages_As_Bot: false
  Forward_From_Bungee: true
````

In summary, the discordSRV works on the survival server, but the 'Forward_From_Bungee' doesn't seem to register. Not sure if it is my config that is incorrect, or what? Let me know if you need more information. Edit: Yes bungee chat works in the standard channel all over, no trouble there.

**Comment by kangarko** [comment]
Are you saying that if you type a message on lobby in standard channel it wont get sent to discord through the survival server?

Make sure there are players on survival server, empty servers wont work.

Set Debug key to [discord] both on lobby and survival. Then please send me the console logs from both servers when you type a message at the nonworking server.

**Comment by SpectreVoid** [comment]
Yes.

This is a message from the lobby read in the Survival console: '[06:01:28 INFO]: [ChatControlRed] [standard] BambiThePenguin: hey' so the message gets there and is read by console, but for some reason is not transmitted to the DiscordSRV. Do the lobby server need the discord channel in the standard or anything related to discord other than using the same chat channel? Cause the messages works fine, just not getting transmitted from the lobby to the discord server, but the survival works just fine and sends it directly to discord as it has DiscordSRV installed. (I had one account in lobby and survival in this example)

![image](https://github.com/kangarko/ChatControl-Red/assets/18666593/cfcf5217-2ff7-44b8-9328-9da16e96d55d)
[06:07:22 INFO]: [ChatControlRed] [standard] BambiThePenguin: Test
[06:07:25 INFO]: [standard] AlbusThePenguin: Test
[06:07:25 INFO]: [discord] [Minecraft > Discord Webhook] From MC 'prat-og-skrik' to Discord 'prat-og-skrik' message: **AlbusThePenguin**: Test

There seem to be a few things not working as intended. The bungee PM system doesn't work either. This may be strangely related to bungeecord. It's enabled , but not working. (The private message system was a conflict with core plugin - fixed)

**Comment by SpectreVoid** [comment]
Solution: I didn't enable the discord settings in lobby as I assumed I didn't need it as it wasn't installed there.
